shenyu-common/src/main/java/org/apache/shenyu/common/concurrent/MemoryLimiter.java

```diff
@@ -214,13 +214,12 @@ public void acquireInterruptibly(final Object o) throws InterruptedException {
         }
         acquireLock.lockInterruptibly();
         try {
-            final long sum = memory.sum();
             final long objectSize = inst.getObjectSize(o);
-            while (sum + objectSize >= memoryLimit) {
+            while (memory.sum() + objectSize >= memoryLimit) {
```

Comment:
hi，` sum + objectSize  `  and  `memory.sum() + objectSize` 

What's the difference?

Reply:
In the while loop, `memory.sum()` gets the latest value.